README.md

```diff
@@ -120,12 +120,18 @@ merge:
   options:
     # "squash" options are only used when the merge method is "squash"
     squash:
+      # "title" defines how the title of the commits message is created when
+      # generating a squash commit. The options are "pull_request_title",
+      # "first_commit_title", and "github_default_title".
+      title: "pull_request_title"
```

Comment:
do we mention that `pull_request_title` and `summarize_commits` are defaults anywhere in the README?

Reply:
Add a line about the default values.